oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/components/filepreview.js

```diff
@@ -0,0 +1,23 @@
+import $ from "jquery";
+
+const iframe = document.querySelector(".fileIframe");
+
+$("#preview-modal").modal({
+  onShow: function () {
+    const contentHeight = window.innerHeight * 0.8;
```

Comment:
This is not responsive, once the modal is shown, it is never resized according to window size changes, possibly overflowing content

Reply:
https://www.w3schools.com/howto/howto_css_responsive_iframes.asp